rows_test.go

```diff
@@ -13,307 +13,3 @@
 // limitations under the License.
 
 package spannerdriver
-
-import (
-	"context"
-	"database/sql"
-	"math"
-	"reflect"
-	"testing"
-)
-
-func TestRowsAtomicTypes(t *testing.T) {
```

Comment:
Integration tests have been moved to integration_test.go and are now skipped if -short has been specified.